src/blockchain-utilities/derivation.ts

```diff
@@ -55,3 +55,25 @@ export const deriveBeaconId = (airnodeAddress: Address, templateId: Hex) =>
 export const deriveBeaconSetId = (beaconIds: Hex[]) =>
   // By convention beacon IDs are sorted alphabetically - the ordering impacts the resulting hash.
   ethers.utils.keccak256(ethers.utils.defaultAbiCoder.encode(['bytes32[]'], [beaconIds]));
+
+export const verifyAirnodeXpub = (airnodeXpub: string, airnodeAddress: Address): ethers.utils.HDNode => {
+  // The xpub is expected to belong to the hardened path m/44'/60'/0'
+  // so we must derive the child default derivation path m/44'/60'/0'/0/0
+  // to compare it and check if xpub belongs to the Airnode wallet
+  const hdNode = ethers.utils.HDNode.fromExtendedKey(airnodeXpub);
+  if (airnodeAddress !== hdNode.derivePath('0/0').address) {
+    throw new Error(`xpub does not belong to Airnode: ${airnodeAddress}`);
+  }
+  return hdNode;
+};
+
+export function deriveSponsorWalletAddress(
```

Comment:
What's with the `export function` and `export const verifyAirnodeXpub` syntax differences?